Source/WebCore/dom/TreeScopeInlines.h

```diff
@@ -30,18 +30,18 @@
 
 namespace WebCore {
 
-inline ContainerNode& TreeScope::rootNode() const
-{
-    return m_rootNode.get();
-}
-
 inline IdTargetObserverRegistry& TreeScope::idTargetObserverRegistry()
 {
     if (m_idTargetObserverRegistry)
         return *m_idTargetObserverRegistry;
     return ensureIdTargetObserverRegistry();
 }
 
+inline ContainerNode& TreeScope::rootNode() const
```

Comment:
Not sure why this one moved.

Reply:
Just a cleanup. It's nice to have each function in its respective class's `Inlines.h`.

Reply:
My point is that you moved it *within* the same `Inlines.h` file it seems.

Reply:
Oh weird. Will revert.